study.md

```diff
@@ -21,18 +21,53 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+Is this correct?: A method is a function that is a property of an object.  Methods can be written as either:
+let obj = {
+  age: function(param1) {
+    code statements;
+  };
+};
+
+or
+
+obj.age(param1) {
+  code statements;
+};
+
+Do both of the above refer to the same method?  Are there any other ways to write a method?
+
+
+
+
 ```
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
+let myFunction = function(param1) {
+  code;
+};
+Is this what a function literal is? Because it has an assignment operator, does that make it a function literal instead of a function (such as obj.function() {} )?
+
+
+
+Is this how callback functions work?
+
+function myFunction() {} // a function is declared
+function callbackFunction(param1, callback) {} // a callback function is declared.  Do you actually write "callback" as the parameter each time you use a callback function?
+callbackFunction(param1, myFunction) {} // when the callback function is called, does the name of the  function to be called back replace the parameter "callback" in callbackFunction?
+
 ```
 
 Thus far how are you feeling about your progress as a developer, please give
 an extensive answer?
 
 ```md
-<!-- your answer here -->
+
+I'm happy with the progress I've made, I obviously know a lot more than I did a week ago.  Though learning javascript can be very frustrating, it is very rewarding when a concept 'clicks' and I understand it.  I really want to fully understand why each part of the language does what it does.  I know that you guys have said that often being a developer means looking stuff up and copy/pasting it, but I would feel much better if I can avoid doing that as much as possible.  I like learning spoken languages, and really understanding what each part of the language means and does.  The same goes for javascript, but I fear that we're moving so fast that I don't have time to fully grasp the fundamentals.  You can't write words and sentences in a spoken language if you don't know the alphabet, and similarly I find it hard to write code if I dont fully understand the basics.
```

Comment:
We encourage people to look things up and learn more on their own, but not necessarily copying and pasting code.  Copying and pasting code should only be done for learning purposes.  e.g., copying, pasting, running, modifying, and examining the modifications and changes.  This isn't a deep course in computer programming, but a primer on the essentials.  Think of it as taking a class in mathematics.  You will only learn the material from parts of the book, but not everything, and we encourage you to explore the material that isn't covered in class.